api/app/src/external/commonwell/organization.ts

```diff
@@ -0,0 +1,71 @@
+import { Organization as CWOrganization } from "@metriport/commonwell-sdk";
+
+import { Config } from "../../shared/config";
+import { commonWellMember, queryMeta, CW_ID_PREFIX } from "../../shared/commonwell";
+import BadRequestError from "../../errors/bad-request";
+import { createOrgId } from "../../shared/oid";
+
+const memberName = "Metriport";
+
+const technicalContact = {
+  name: "Metriport Team",
+  title: "Engineering",
+  email: "support@metriport.com",
+  phone: "(415)-941-3282",
+};
```

Comment:
as per @leite08's comment - do we at least have a ticket to make this configurable?

don't want to hardcode this for folk that will be self-hosting